doc/src/build/wallet.md

```diff
@@ -162,33 +212,35 @@ The following commands are supported by the wallet:
     `transfer`       Transfer an object
 Use `help <command>` to see more information on each command.
 
-The wallet can be started in two modes: interactive shell or command line interface.
+You can start the wallet in two modes: interactive shell or command line interface.
 
 ### Interactive shell
 
-To start the interactive shell, execute the following (in a different terminal window than one used to execute `sui start`):
+To start the interactive shell, execute the following (in a different
+terminal window than one used to execute `sui start`). Assuming you
```

Comment:
This is now somewhat redundant with the first paragraph of the "Using the wallet" section